TextToSpeech/TextToSpeech.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Isn't this auto-generated? probably not needed to be checked in.

Reply:
Removed the TextToSpeech.json.